src/main/java/com/TableTalk/Enterprise/dto/ProfilePicture.java

```diff
@@ -2,22 +2,13 @@
 
 import lombok.Data;
 
-//creating a profile picture class for efficiency. Might change it to a generic Photo class if we need photos anywhere else.
-
-// @Entity
-// @Table(name="photos")
```

Comment:
No comments since we're using version control. I know you're using them for debugging and I do it too, but it is bad practice when using version control.